### ISSUE TITLE
Can't use Fb2Utils

### ISSUE THREAD

**Issue by eskander123123** [bogdandonduk/FileParsersToolbox#1]
Hi bro, I want to use your code for parsing fb2 books for an android application, but my Fb2Utils element is highlighted in red and I cannot use it, although I did everything according to the instructions, could you help me? my mail: mreska07@mail.ru

**Comment by bogdandonduk** [comment]
Sure. Show me some more info (screenshots). Text me in telegram if you have one @bogdandonduk

**Comment by bogdandonduk** [comment]
Please write me there @bogdandonduk

ср, 15 сент. 2021 г., 12:00 Богдан Дондук ***@***.***>:

**Comment by bogdandonduk** [comment]
Hi man. Sure. Got a telegram?

вт, 14 сент. 2021 г., 22:00 eskander123123 ***@***.***>:

**Comment by eskander123123** [comment]
Yes, bro, let's go to Microsoft Teams, or what will be more convenient for you?
